resources/tests/EasyCrypt_GenDoc_Tutorial.ec

```diff
@@ -0,0 +1,311 @@
+(*^
+  EasyCrypt_GenDoc_Tutorial.ec
+
+  To generate documentation for a source file, run the following command:
+  {{
+  <easycrypt-executable> gendoc [-outdir <output-directory>] <source-file>
```

Comment:
This is docgen, now. That was particularly unhelpful to my caffeine-deprived brain cell.

Reply:
Damn, that's actually disgusting I forgot to change that. Sorry.